samples/browser/README.md

```diff
@@ -12,6 +12,18 @@ The browser app uses a product flavor:
 
 * **channel**: Using different release channels of GeckoView: _nightly_, _beta_, _production_. In most cases you want to use the _nightly_ flavor as this will support all of the latest functionality.
 
+## Glean SDK support
+
+This sample application comes with Glean SDK telemetry initialized by default, but with upload disabled (no data is being sent).
+This is for creating a simpler metric testing workflow for Gecko engineers that need to add their metrics to Gecko and expose them to Mozilla mobile products.
+See [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1592935) for more context.
+
+In order to enable data upload for testing purposes, change the `Glean.setUploadEnabled(false)` to `Glean.setUploadEnabled(true)` in [`SampleApplication.kt`](src/main/java/org/mozilla/samples/browser/SampleApplication.kt).
+
+Glean will send metrics from any Glean-enabled component used in this sample application.
+
+Data review for enabling the Glean SDK for this application can be found [here]().
```

Comment:
TODO: add a link to the data review response here once [data review](https://bugzilla.mozilla.org/show_bug.cgi?id=1592935#c5) is cleared.